semgrep_output_v1.atd

```diff
@@ -245,6 +245,9 @@ type core_error_kind
   | FatalError (* missing file, OCaml errors, etc. *) <json name="Fatal error">
   | Timeout <json name="Timeout">
   | OutOfMemory <json name="Out of memory">
+  (* pro-engine specific *)
+  | TimeoutDuringPreprocessing <json name="Timeout during preprocessing">
```

Comment:
What is preprocessing in semgrep-pro? When we build the SASTs and SIGs for every files?